tests/integration/transactions/test_delete_oracle.py

```diff
@@ -0,0 +1,59 @@
+import time
+
+from tests.integration.integration_test_case import IntegrationTestCase
+from tests.integration.it_utils import (
+    sign_and_reliable_submission_async,
+    test_async_and_sync,
+)
+from tests.integration.reusable_values import WALLET
+from xrpl.models import AccountObjects, AccountObjectType, OracleDelete, OracleSet
+from xrpl.models.response import ResponseStatus
+from xrpl.models.transactions.oracle_set import PriceData
+from xrpl.utils import str_to_hex
+
+_PROVIDER = str_to_hex("chainlink")
+_ASSET_CLASS = str_to_hex("currency")
+
+
+class TestDeleteOracle(IntegrationTestCase):
+    @test_async_and_sync(globals())
+    async def test_basic(self, client):
+        oracle_id = self.value
+
+        # Create PriceOracle, to be deleted later
+        tx = OracleSet(
+            account=WALLET.address,
+            # unlike the integration tests for OracleSet transaction, we do not have to
+            # dynamically change the oracle_document_id for these integration tests.
+            # This is because the Oracle LedgerObject is deleted by the end of the test.
+            oracle_document_id=oracle_id,
+            provider=_PROVIDER,
+            asset_class=_ASSET_CLASS,
+            last_update_time=int(time.time()),
```

Comment:
Shouldn't this be converted to ripple epoch time? `int(time.time())` gets current unix timestamp which isn't the correct format this transaction requires.

Reply:
no, the input needs to be unix time. I've requested Rome to update the docs.

Reply:
Ah ok, the spec states this too which also should be updated - https://github.com/XRPLF/XRPL-Standards/discussions/129